custom_dc/custom_data.md

```diff
@@ -45,21 +45,21 @@ San Francisco,2023,300,300,200,50
 San Jose,2023,400,400,300,0  
 ```
 
-The _ENTITY_ is an existing property in the Data Commons knowledge graph that is used to describe an entity, most commonly a place. The best way to think of the entity type is as a key that could be used to join to other data sets. The column heading can be expressed as any existing place-related property; see [Place types](../place_types.md) for a full list. It may also be any of the special DCID prefixes listed in (Special place names)[#special-names].
+The _ENTITY_ is an existing property in the Data Commons knowledge graph that is used to describe an entity, most commonly a place. The best way to think of the entity type is as a key that could be used to join to other data sets. The column heading can be expressed as any existing place-related property; see [Place types](../place_types.html) for a full list. It may also be any of the special DCID prefixes listed in (Special place names)[#special-names].
 
```

Comment:
`(Special place names)[#special-names]` is not rendering as a link for me. I think you mean to swap the parentheses and square braces.
```suggestion
The _ENTITY_ is an existing property in the Data Commons knowledge graph that is used to describe an entity, most commonly a place. The best way to think of the entity type is as a key that could be used to join to other data sets. The column heading can be expressed as any existing place-related property; see [Place types](../place_types.html) for a full list. It may also be any of the special DCID prefixes listed in [Special place names](#special-names).
```